### PR TITLE
Camel 3.16 whats new blog.

### PR DESCRIPTION
**Pull request by davsclaus**

### CODE REVIEW

**Comment by davsclaus** [comment]
@orpiske @oscerd @essobedo  this blog post needs more work. Feel free to update and add other stuff too.

**Comment by essobedo** [comment]
Hi no problem, I will complete the part on which I worked asap @davsclaus when do you want to publish it?

**Comment by davsclaus** [comment]
Oh at first we need gregor to release 3.16.0 then we can publish this afterwards. So I think on tuesday, or maybe monday afternoon.

File: content/blog/2022/03/camel316-whatsnew/index.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+---
+
+Apache Camel 3.16 has just been [released](/blog/2022/03/RELEASE-3.16.0/).
+
+This release introduces a set of new features and noticeable improvements that we will cover in this blog post.
+
+### Resume from Offset
+
+TODO:
+https://camel.apache.org/blog/2022/03/resume-api-v2/
```
Comment (orpiske) [inline content/blog/2022/03/camel316-whatsnew/index.md:16]: This release brings a new API to simplify consuming data at scale: the resume API V2. Please check the [blog post](https://camel.apache.org/blog/2022/03/resume-api-v2/) we wrote to introduce it to our community.

File: content/blog/2022/03/camel316-whatsnew/index.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+TODO: Work started on marking up all headers in every components for automatic documentation, and tooling support.
+
+### Camel Kafka
+
+TODO: 
+
+We added specific health-checks for kafka producer and consumers that checks the kafka-client
+internals if the connectivity with the kafka brokers is healthy or not.
+
+We did other bug fixes, and made camel-kafka more roboust.
```
Comment (orpiske) [inline content/blog/2022/03/camel316-whatsnew/index.md:87]: Maybe we can add something like: "we continue to make the camel-kafka component more robust. In this release we fixed offset management issues. Please check the release notes for details".

File: content/blog/2022/03/camel316-whatsnew/index.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+
+We have upgraded to the latest Spring Boot 2.6 release.
+
+Added more unit tests to various starter JARs to increase the QA of this project.
+
+## Upgrading
+
+Make sure to read the [upgrade guide](/manual/camel-3x-upgrade-guide-3_16.html) if you are upgrading from a previous Camel version.
+
+## Release Notes
```
Comment (orpiske) [inline content/blog/2022/03/camel316-whatsnew/index.md:103]: Nice one @davsclaus. I'd also include the deprecation notice for the `camel-testcontainers*` components and a link to the blog post: https://camel.apache.org/blog/2022/02/camel-test-infra/

Probably something along the lines of: 
```
## Deprecated Components: Camel Testcontainers

In this release we are replacing the camel-testcontainers components with the camel-test-infra. These new set of test-related components simplify integration testing with Camel, reduce our maintenance effort, allow us to test edge case scenarios more easily and more. Please check the [blog post](https://camel.apache.org/blog/2022/02/camel-test-infra/) where we explained about the changes.

```

**Review by orpiske — APPROVED** [review]
I added some suggestions, but it LGTM.

**Comment by orpiske** [comment]
BTW: I can do the changes myself, if you are fine w/ it.

**Comment by davsclaus** [comment]
yeah please go ahread and also add your names to the author list, we are doing this blog together

**Review by gzurowski — APPROVED** [review]
LGTM

**Comment by essobedo** [comment]
@davsclaus I added some more details, let me know if expected more or something else

**Comment by davsclaus** [comment]
@essobedo great thanks. Maybe add a little test about how far we got with the headers, was it not A..G or something ?

**Comment by essobedo** [comment]
@davsclaus You mean that you would like me to add a not about the current progress?

**Comment by essobedo** [comment]
@davsclaus I've added it as footnote.

**Comment by davsclaus** [comment]
@oscerd do you want to add a little blurb in this post where there is a TODO that refers to your work on the secrets stuff

**Comment by oscerd** [comment]
Sure

**Comment by oscerd** [comment]
Done.

**Comment by essobedo** [comment]
I don't know if it is only a local problem or not but the code snippets are hardly readable as you can see below:

image
